automated-comments/java/hamming/avoid_character_literals.md

```diff
@@ -0,0 +1,3 @@
+This solution used character literals (eg. `'T'`), but they are not necessary for this exercise.
+Although the description mentions the DNA nucleotide use-case, you can calculate the Hamming distance for any two strings of the same length.
+Check if the letters are different, regardless of what the letters are.
```

Comment:
potentially start with this. From the [guidelines](https://github.com/exercism/automated-analysis/blob/5b01bae1faced4c771308bab10a4fd6277e2a4d1/docs/comment_copy_guidelines.md):

> When possible, put the recommendation first, then the explanation.